### DISCUSSION TITLE
Couple of questions about project

### DISCUSSION THREAD

**Discussion by Shadowic-ai** [Owlet7/wadfusion#37] [Q&A] [answered]
1. Why NoJump and NoCrouch are in MAPINFO, but not also NoFreelook?
2. How would episodic campaigns work? Like, would it just start with later map and if started with earlier map will just continue after text, or would it end like UD's episodes? I suspect it's first, because second would require pictures, but... just want to hear your ideas.

**Comment by Owlet7** [comment, marked as answer]
1. Playing Doom with freelook is quite popular. While I don't really like it myself, forcing it off would alienate a lot of users.
2. By just warping to a later level in the campaign. Nothing else will change. The episodes themselves are based on DOOM II's SETUP.EXE

![image](https://github.com/user-attachments/assets/33cf3e97-c92a-43aa-94fa-ca9452775825)
